modules/dateUtils.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
+
+export function getCurrentDate() {
```

Comment:
- [ ] Nice work implementing the `getCurrentDate` :clap: However it would be nice to use an **arrow** syntax since this activity is about ES6 and also to follow the project requirement (see Image)
![Screenshot from 2023-06-14 22-57-39](https://github.com/Eccb7/AwesomeBooks_ES6/assets/91142432/384af5a5-4db4-44bb-a47f-f44bb172db00)